sdk/deepbook-v3/src/types/index.ts

```diff
@@ -90,4 +87,4 @@ export interface Config {
 	DEEP_TREASURY_ID: string;
 }
 
-export type Environment = 'mainnet' | 'testnet';
+export type Environment = 'mainnet' | 'testnet' | 'devnet' | 'localnet';
```

Comment:
As far as I can tell this is only used for selecting which config to use, and we only have configs for mainnet and testnet, so adding the other envs here will allow users to specify unsupported envs

Reply:
Good point, agree here